### PR TITLE
[WIP] Switch to bundle image and operator-sdk

### PR DESCRIPTION
**Pull request by clobrano**
Replace operator workflow + index image with bundle image and operator-sdk.

### CODE REVIEW

File: ci-operator/config/medik8s/machine-deletion-remediation/medik8s-machine-deletion-remediation-main.yaml [hunk 1]
```diff
@@ -20,9 +24,10 @@ operator:
 releases:
   latest:
     candidate:
+      architecture: amd64
       product: ocp
-      stream: ci
-      version: "4.10"
+      stream: nightly
+      version: "4.11"
```
Comment (razo7) [inline ci-operator/config/medik8s/machine-deletion-remediation/medik8s-machine-deletion-remediation-main.yaml:30]: Why only OCP v4.11?

Comment (clobrano) [reply ci-operator/config/medik8s/machine-deletion-remediation/medik8s-machine-deletion-remediation-main.yaml:30]: That was a mistake, I'll revert back to 4.10

**Pull request closed without merging**